--- CMakeLists.txt	Fri Dec 16 06:06:17 2016
+++ CMakeLists.txt	Fri Dec 16 06:10:11 2016
@@ -27,7 +27,7 @@
 # setup target
 ogre_add_library(OgreOverlay ${OGRE_COMP_LIB_TYPE} ${CONAN_LIBS_BZIP2} ${CONAN_LIBS_LIBPNG} ${HEADER_FILES} ${SOURCE_FILES} ${PLATFORM_HEADER_FILES} ${PLATFORM_SOURCE_FILES})
 set_target_properties(OgreOverlay PROPERTIES VERSION ${OGRE_SOVERSION} SOVERSION ${OGRE_SOVERSION})
-target_link_libraries(OgreOverlay OgreMain ${FREETYPE_LIBRARIES})
+target_link_libraries(OgreOverlay OgreMain ${FREETYPE_LIBRARIES} ${CONAN_LIBS_BZIP2} ${CONAN_LIBS_LIBPNG})
 if (OGRE_CONFIG_THREADS)
   target_link_libraries(OgreOverlay ${OGRE_THREAD_LIBRARIES})
 endif ()
