--- CMakeLists.txt	2018-04-24 22:08:34.861084700 +0200
+++ CMakeLists.txt	2018-04-24 22:54:35.961230900 +0200
@@ -48,9 +48,12 @@
 endif ()
 
 project(OGRE)
+include(../conanbuildinfo.cmake)
+CONAN_BASIC_SETUP()
 
 # Include necessary submodules
 set(CMAKE_MODULE_PATH
+  ${CONAN_CMAKE_MODULE_PATH}
   "${OGRE_SOURCE_DIR}/CMake"
   "${OGRE_SOURCE_DIR}/CMake/Utils"
   "${OGRE_SOURCE_DIR}/CMake/Packages"
@@ -128,12 +131,10 @@
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /fp:fast")
   # Enable intrinsics on MSVC in debug mode
   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /Oi")
-  if (CMAKE_CL_64)
-    # Visual Studio bails out on debug builds in 64bit mode unless
+    # Visual Studio bails out on debug builds unless
 	# this flag is set...
 	set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /bigobj")
 	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} /bigobj")
-  endif ()
 
   if (OGRE_PROJECT_FOLDERS)  
     # Turn on the ability to create folders to organize projects (.vcproj)
