--- OgreD3D9EngineDll.cpp	Mon Dec 12 07:49:24 2016
+++ OgreD3D9EngineDll.cpp	Mon Dec 12 16:26:04 2016
@@ -28,6 +28,10 @@
 #include "OgreRoot.h"
 #include "OgreD3D9Plugin.h"
 
+#include <Windows.h>
+#include <stdio.h>
+int (WINAPIV * __vsnprintf)(char *, size_t, const char*, va_list) = _vsnprintf;
+
 #ifndef OGRE_STATIC_LIB
 
 namespace Ogre 
@@ -46,4 +50,4 @@
 		OGRE_DELETE plugin;
 	}
 }
-#endif
\ No newline at end of file
+#endif
