--- InstallDependencies.cmake	Sat Dec 17 18:31:45 2016
+++ InstallDependencies.cmake	Sat Dec 17 19:53:35 2016
@@ -132,12 +132,6 @@
 endif () # OGRE_INSTALL_DEPENDENCIES
     
   if(WIN32)
-    # copy the dependency DLLs to the right places
-    if(NOT OGRE_BUILD_PLATFORM_WINRT)
-      install_debug(OIS_d.dll)
-      install_release(OIS.dll)
-    endif ()
-
     if (OGRE_BUILD_PLUGIN_CG)
       # if MinGW or NMake, the release/debug cg.dll's would conflict, so just pick one
       if (MINGW OR (CMAKE_GENERATOR STREQUAL "NMake Makefiles"))
